build.gradle.kts

```diff
@@ -0,0 +1,32 @@
+plugins {
```

Comment:
Please use Groovy and tabs

Reply:
And trailing newline!

Reply:
a60365616650e6f8b4b1d17c5b624348ee83406e